Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -221,7 +233,10 @@ final class DeclarationTests: XCTestCase {
       "_ = foo/* */?.description1️⃣",
       diagnostics: [
         DiagnosticSpec(message: "expected ':' and expression after '? ...' in ternary expression", fixIts: ["insert ':' and expression"])
-      ]
+      ],
+      fixedSource: """
+        _ = foo/* */?.description: <#expression#>
```

Comment:
Inside ternary expressions, `:` should have a space on either side.

Reply:
https://github.com/apple/swift-syntax/issues/1607